assets/lib/commands/out.rb

```diff
@@ -83,6 +83,14 @@ def output
         metadata << { 'name' => 'label', 'value' => params.label }
```

Comment:
Should probably change `params.label` here to `label` for consistency.

Reply:
For strings, putting in label would return `[label]` instead of `label`. I was worried that it would break expected backwards compatibility. Is that a concern?

Reply:
That's what I meant - worth making it consistent with the new behavior.  It's _technically_ a break, but it's a _really small_ one - the only thing that does is print out metadata in concourse.

Reply:
Done. (Tests pass too)